.buildkite/pipeline.yml

```diff
@@ -597,6 +597,29 @@ steps:
               artifact_paths:
                 - "perf8-report-*/**/*"
 
+  - group: ":building_construction: Build Python Packages"
```

Comment:
Should we do the build after all steps succeed instead of running in parallel? How long does the step take normally?